descartes_trajectory/test/cart_trajectory_pt.cpp

```diff
@@ -77,7 +77,7 @@ TEST(CartTrajPt, getPoses)
   std::vector<std::vector<double> >joint_solutions;
 
   ROS_INFO_STREAM("Testing fuzzy pos point");
-  CartesianRobot robot(POS_TOL+2*EPSILON, ORIENT_TOL+2*EPSILON);
+  CartesianRobot robot(POS_TOL+2*EPSILON, 2*M_PI+2*EPSILON);
```

Comment:
Well as long as you're okay with me changing the discretization size. Right now it'll take ~ 30 * 30 * 30 * 10 * 10 * 10 = 27 million samples for a given point.